packages/integration/src/bitbucket/core.test.ts

```diff
@@ -249,6 +251,42 @@ describe('bitbucket core', () => {
         config,
       );
       expect(defaultBranch).toEqual('main');
-    });
+      });
+
+      it('return default branch for Bitbucket Server for bitbucket version 5.11', async () => {
```

Comment:
Let's just say this?

```suggestion
      it('return default branch for Bitbucket Server for older bitbucket versions < 7.5', async () => {
```

Looking at the reference docs, I can see `default-branch` in 7.5 but not in earlier versions.
https://docs.atlassian.com/bitbucket-server/rest/7.5.0/bitbucket-rest.html#idp202
https://docs.atlassian.com/bitbucket-server/rest/7.4.0/bitbucket-rest.html